composer_triggering_dags_with_functions_and_pubsub/main.tf

```diff
@@ -0,0 +1,217 @@
+# Copyright 2023 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START triggering_dags_with_functions_and_pubsub]
+#   Triggering DAGs Using Cloud Function and Pub/Sub Messages with Terraform
+#
+#   Usage:
+#       1. Select or create a new project that you will use to create the resources
+#       2. Upload into your directory the pubsub_function.zip file: https://github.com/akvelon/python-docs-samples/tree/main/composer_tutorials/triggering-dags-with-functions-and-pubsub/cloud_function
```

Comment:
These links are currently page not found 404. Is this expected?

I think terraform-docs-samples tests will fail without the referenced files included. Can you include them in the same directory with main.tf, similar to the way we've done so [here](https://github.com/terraform-google-modules/terraform-docs-samples/tree/main/cloudfunctions2_basic_gcs)? Or is important that the zip and source code be single-sourced?